samples/sensor/mcux_lpcmp/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _mcux_lpcmp:
+
+NXP MCUX Low-power Analog Comparator (LPCMP)
+############################################
+
+Overview
+********
+
+This sample show how to use the NXP MCUX Analog Comparator (LPCMP) driver. The
+sample supports the :ref:`frdm_mcxn947`.
+
+The input voltage for the negative input of the analog comparator is provided
+by the LPCMP internal Digital-to-Analog Converter (DAC). The input voltage for
+the positive input can be connected to an external voltage supply, the voltage
+signal is captured on J2-17, need change the external voltage signal to check the
+output.
+
+The output value of the analog comparator is reported on the console.
+
+Building and Running
+********************
+
+Building and Running for NXP frdm_mcxn947
```

Comment:
should be generically worded I think, you can keep frdm mcx in the build command as an example